.github/workflows/build-test-fedora.yml

```diff
@@ -94,6 +94,7 @@ jobs:
           MR_CMAKE_OPTIONS: >
             -DMR_USE_CPP_23=ON
             -DMR_PCH_USE_EXTRA_HEADERS=ON
+            -DMESHLIB_BUILD_MRMESH_PY_LEGACY=ON
```

Comment:
why ON?

Reply:
Because we don't run MRBind on Fedora, simply because I haven't made the scripts to do so. So on Fedora I use the legacy bindings.